.github/workflows/ci_tests.yaml

```diff
@@ -138,3 +141,66 @@ jobs:
           file: ./coverage.xml # optional
           env_vars: OS,PYTHON
           fail_ci_if_error: true
+
+      - name: Checkout the gh-pages-testing branch
+        uses: actions/checkout@28c7f3d2b5162b5ddd3dfd9a45aa55eaf396478b
+        with:
+          ref: gh-pages-testing
```

Comment:
Use `gh-pages-testing` branch for testing purpose. Need to revert the branch name to `gh-pages` before merging.